javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,4 @@
+let yearOfBirth = 1987;
+let yearFuture = 2027;
+let age = yearFuture-yearOfBirth;
```

Comment:
Good job. I just recommend you to use `const` not `let` here as long as you don't change the value of these variables.

Reply:
Changed to Const